src/osmet-pack

```diff
@@ -6,24 +6,23 @@ if [ ! -f /etc/cosa-supermin ]; then
     # shellcheck source=src/cmdlib.sh
     . "${dn}"/cmdlib.sh
 
-    img_src=$1; shift
+    img_src=$(realpath "$1"); shift
     sector_size=$1; shift
-    osmet_dest=$1; shift
+    osmet_dest=$(realpath "$1"); shift
     checksum=$1; shift
-    speed=$1; shift
```

Comment:
We're losing the `speed` bits here, so I think CI will misinterpret the next argument (`coreinst`) as `--fast`.

Reply:
oh...I took your `osmet` entirely. Fixed up.

Reply:
There are a few other spots that use `speed` too. It's also adding back some `udevadm`-related bits we had removed. Probably cleaner to either drop all of that for now (we don't have to support parallel everything in one shot). Or if you'd really like, rebase that patch first on the latest master.

Reply:
Yeah, let's drop it.

Reply:
Dropped.